homeassistant/components/garages_amsterdam/binary_sensor.py

```diff
@@ -43,39 +40,22 @@ async def async_setup_entry(
 class GaragesamsterdamBinarySensor(CoordinatorEntity, BinarySensorEntity):
     """Binary Sensor representing garages amsterdam data."""
 
+    _attr_attribution = ATTRIBUTION
```

Comment:
I think we can also apply this in the `sensor.py`?

Reply:
Done